portfolio/src/main/java/com/google/sps/servlets/MyServlet.java

```diff
@@ -0,0 +1,17 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("one thing about me")
```

Comment:
I think it's best to have web page URLs as one word. So if you want multiple words, you would want to connect them with a "-" or "_". So something like "one-thing-about-me" would be best here

Reply:
Nevermind, I see you updated this in the next commit. nice!